### PR TITLE
update user profile

### PR DESCRIPTION
**Pull request by vietanhbkaaa**

### CODE REVIEW

**Comment by nhs3108** [comment]
OK

**Comment by ThuBM** [comment]
OK

**Comment by ThuBM** [comment]
OK

File: config/routes.rb [hunk 1]
```diff
@@ -1,4 +1,8 @@
 Rails.application.routes.draw do
+  get "users/edit"
```
Comment (chuanhtuan) [inline config/routes.rb]: remove

File: config/routes.rb [hunk 2]
```diff
@@ -1,4 +1,8 @@
...
+
+  get "users/update"
```
Comment (chuanhtuan) [inline config/routes.rb]: remove

**Pull request merged**